resources/scripts/root/fix-serialport.js

```diff
@@ -23,5 +26,25 @@ if (fs.existsSync(binFile)) {
     console.error('Problem placing pre-compiled binary.', e);
   }
 } else {
-  console.log('Skipping placing pre-compiled serialport binary, unupported OS/architechture.');
+  var npmArgs = [
+    'install', '--runtime=electron',
+    '--disturl=https://atom.io/download/atom-shell', '--target=1.0.2',
+    'serialport'];
+
+  console.log('Unable to place pre-compiled serialport binary, unupported OS/architechture.');
```

Comment:
This is good! I'm guessing this will only fail on Linux 32bit and Arm. Of course Chromium is actually unsupported for both of those! So.. this is going to be darn rare.